examples/sdp3x_default_address/sdp3x_default_address.ino

```diff
@@ -9,33 +9,32 @@ void setup() {
   Wire.begin();
   Serial.begin(9600);
   delay(1000); // let serial console settle
-
-  int ret = sdp.init();
-  if (ret == 0) {
-      Serial.print("init(): success\n");
-  } else {
+  do {
+    int ret = sdp.init();
+    if (ret == 0) {
+    Serial.print("init(): success\n");
```

Comment:
The content of the `if` block is not indented properly. Please shift accordingly.